template.yaml

```diff
@@ -166,6 +171,8 @@ Conditions:
 Globals:
   Function:
     Timeout: 60
+    PermissionsBoundary: !If [CreateDefaultBoundaryPolicy, !Ref DefaultPermissionBoundaryPolicy, !Ref PermissionBoundaryPolicyArn]
+    Runtime: nodejs12.x
```

Comment:
I know this is an unrelated change but since you are touching this (and testing it), could we go to Node 14 while we're at it? (unless you don't have the time to test that)

Reply:
@ottokruse should be all fixed now - and tested thanks